src/microbe_stage/Spawners.cs

```diff
@@ -73,10 +72,9 @@ public static class SpawnHelpers
             for (int i = 0; i < random.Next(Constants.MIN_BACTERIAL_COLONY_SIZE,
                 Constants.MAX_BACTERIAL_COLONY_SIZE + 1); i++)
             {
-                // Dont spawn them on top of each other because it
-                // causes them to bounce around and lag
-                entities.Add(SpawnMicrobe(species, location + curSpawn, worldRoot, microbeScene, true,
-                    cloudSystem, currentGame));
+                SpawnDelayedBacteria(species, location + curSpawn, worldRoot, microbeScene,
```

Comment:
I think it would be really nice if this code could be made into a C# enumerator, so that these spawns could be made using `yield return`